### ISSUE TITLE
Switch away from backbone

### ISSUE THREAD

**Issue by danvk** [hammerlab/pileup.js#134]
It's ~60k of JS, which seems like a lot given that we're only using it for `.on()`, `.off()` and `.trigger()`.

**Comment by danvk** [comment]
I think it's actually more than 60k since Backbone also pulls in jQuery.

**Comment by danvk** [comment]
I tried this out.

Before:

    4458583 dist/pileup.js
     587942 dist/pileup.min.js

After:

    4326554 dist/pileup.js
     570800 dist/pileup.min.js

So we save 17k = 3% on the minified code and 132k (also 3%) on the unminified code.

I'm not sure why the savings is over 2x the size of backbone (`backbone.js` is only 60k). But it's not a huge win.

I've pushed the work to [a branch][1] if we want to follow up on this.

[1]: https://github.com/hammerlab/pileup.js/tree/drop-backbone
